### PR TITLE
Tracking for Agent-based installer interactions and others

### PR DESCRIPTION
**Pull request by jgyselov**
https://issues.redhat.com/browse/MGMT-9701
https://issues.redhat.com/browse/MGMT-13606

Related to: https://gitlab.cee.redhat.com/service/uhc-portal/-/merge_requests/3908

### CODE REVIEW

File: data/openshift.yml [hunk 1]
```diff
@@ -424,3 +445,60 @@ openshift:
...
+        - '{} [data-ouia-component-id="baremetal"] button:contains("Download pull secret")'
+    "Clusters - Bare Metal: Local Agent-based - Copy pull secret":
+      selectors:
+        - '{} [data-ouia-component-id="baremetal"] button:contains("Copy pull secret")'
+    "Clusters - Bare Metal: Local Agent-based - Get started":
+      selectors:
+        - '{} [data-ouia-component-id="baremetal"] a:contains("Get started")'
+    "Clusters - vSphere: Local Agent-based - Download installer":
+      selectors:
+        - '{} [data-ouia-component-id="vsphere"] a:contains("Download installer")'
```
[the remark this answers is not in the document]
Comment (jkilzi) [reply data/openshift.yml:483]: 10x @celdrake. It was handled in 34a857eb22d76dac1536c74acb9c300c3686850b

[the remark this answers is not in the document]
Comment (jgyselov) [reply data/openshift.yml:492]: We synced on the issue, it's on my to-do list.

File: data/openshift.yml [hunk 2]
```diff
@@ -439,3 +454,60 @@ openshift:
...
+    "Cluster create - ARM Bare Metal - User provisioned":
+      selectors:
+        - '{} [data-ouia-component-id="arm-instructions-chooser"] #select-full-control'
+    "Clusters - Datacenter: Run Agent-based Installer locally":
+      selectors:
+        - '{} a:contains("Run Agent-based Installer locally")'
+    "Clusters - Bare Metal: Local Agent-based":
+      selectors:
+        - '{} [data-ouia-component-id="metal-instructions-chooser"] #select-agent-based'
+    "Clusters - vSphere: Local Agent-based"
```
Comment (ryelo) [inline data/openshift.yml:472]:
```suggestion
    "Clusters - vSphere: Local Agent-based":
```

File: data/openshift.yml [hunk 3]
```diff
@@ -439,3 +454,60 @@ openshift:
...
+      selectors:
+        - '{} [data-ouia-component-id="vsphere-instructions-chooser"] #select-agent-based'
+    "Clusters - Platform Agnostic: Local Agent-based"
```
Comment (ryelo) [inline data/openshift.yml:475]:
```suggestion
    "Clusters - Platform Agnostic: Local Agent-based":
```

**Review by ryelo — CHANGES_REQUESTED** [review]
Few linting errors

File: data/openshift.yml (hunk 3 above)
Comment (jgyselov) [reply data/openshift.yml]: Fixed, thank you!

**Pull request merged**